book/website/communication/binder/zero-to-binder.md

```diff
@@ -20,6 +20,7 @@ You can always refresh the window if you see the "... is taking longer to load,
 If you are following the R path, we have included some alternative steps using the [`holepunch` package](https://github.com/karthik/holepunch) which will build your environment using a [rocker base image](https://github.com/rocker-org/rocker) and should, ultimately, be faster to build and launch.
 ```
 
+
```

Comment:
Is this intended?

Reply:
Don't know -- I assume, Arya can explain this.

Reply:
No intentional edits were made here. There was a version conflict. The main project uses Jupyter Book version 0.13, but the pathway feature [developed here](https://github.com/alan-turing-institute/bio-Turing-Way/tree/main) requires version 0.11.2. While dealing with the conflict, this edit accidentally became a part of it.

Reply:
```suggestion
```